src/select/SimpleSelect.js

```diff
@@ -240,10 +240,13 @@ class SimpleSelect extends Component {
 
   getFocusedOption(direction, options) {
     const isOpen = this.props.isOpen || this.state.isOpen
-    const { valueKey } = this.props
+    const { allowSelectAll, allOption, valueKey } = this.props
     const { focusedOption } = this.state
+
+    const flatOptions = flattenOptions(options, allowSelectAll, allOption)
```

Comment:
Does flat imply "ungrouped"? Just making sure I understand the nomenclature.

Reply:
Yes, ungrouped and without the group/header level options.